src/main/java/ch/njol/skript/classes/ClassInfo.java

```diff
@@ -33,51 +35,37 @@ public class ClassInfo<T> implements Debuggable {
 	private final String codeName;
 	private final Noun name;
 
-	@Nullable
-	private DefaultExpression<T> defaultExpression = null;
-
-	@Nullable
-	private Parser<? extends T> parser = null;
+	private @Nullable DefaultExpression<T> defaultExpression = null;
 
-	@Nullable
-	private Cloner<T> cloner = null;
+	private @Nullable Parser<? extends T> parser = null;
 
-	@Nullable Pattern[] userInputPatterns = null;
+	private @Nullable Cloner<T> cloner = null;
 
-	@Nullable
-	private Changer<? super T> changer = null;
+	private Pattern @Nullable [] userInputPatterns = null;
```

Comment:
merge mistake, this needs to be package private
```suggestion
	Pattern @Nullable [] userInputPatterns = null;
```